### DISCUSSION TITLE
Some colours work, others do not. Tailwind v3.0.23

### DISCUSSION THREAD

**Discussion by dwedigital** [tailwindlabs/tailwindcss#8073] [Help]
I am using tailwind as per the laravel install guide but made some tweaks as using a different framework.

When I do `bg-yellow-500` on a div the colour changes as expected however not all of the standard colour palette works e.g. `bg-slate-500`

Link to the repo I am working on (use the `develop` branch):

[Repo](https://github.com/dwedigital/wagerv2)

**Comment by alexanderbluhm** [comment]
Hi @dwedigital The repo link doesn't work.

**Comment by dwedigital** [comment]
Hi @alexanderbluhm I accidentally did this as a discussion and replicated as an issue which I closed since figuiring out the issue. Repo is now private.
